<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PingWidget</class>
 <widget class="QWidget" name="PingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>125</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ping</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPingAddress">
       <property name="text">
        <string>Address</string>
       </property>
       <property name="buddy">
        <cstring>cmbPingHost</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbPingHost">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhUrlCharactersOnly</set>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkPingDontFragment">
       <property name="text">
        <string>Don't fragment</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPingTimeout">
       <property name="text">
        <string>Timeout</string>
       </property>
       <property name="buddy">
        <cstring>sbPingTimeout</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPingTimeout">
       <property name="maximum">
        <number>9900</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPingPacketSize">
       <property name="text">
        <string>Packet size</string>
       </property>
       <property name="buddy">
        <cstring>sbPingPacketSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPingPacketSize">
       <property name="maximum">
        <number>64000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPingPackets">
       <property name="text">
        <string>Packets</string>
       </property>
       <property name="buddy">
        <cstring>sbPingPackets</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPingPackets"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="linePing">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPingStatusBad">
       <property name="text">
        <string>Status is bad when</string>
       </property>
       <property name="buddy">
        <cstring>sbPingStatusBad</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPingStatusBad">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPingPacketsLost">
       <property name="text">
        <string>(or more) % of packets lost</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPingDisplay">
       <property name="text">
        <string>Display</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="rbPingDisplayTime">
       <property name="text">
        <string>Reply time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbPingDisplayLost">
       <property name="text">
        <string>% of lost</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbPingDisplayReceived">
       <property name="text">
        <string>% of received</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
